### PR TITLE
lastSeenOn in seconds

### PR DESCRIPTION
**Pull request by DrewJohnsonGT**
# Description

lastSeenOn was in seconds and being processed in ms

## Type of change

Please leave any irrelevant options unchecked.

- [x] Bug fix (non-breaking change which fixes an issue)

### General Development Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### Integration Development Checklist:

Please leave any irrelevant options unchecked.

- [ ] I have checked for additional permissions required to call any new API
      endpoints, and have documented any additional permissions in
      `jupiterone.md`, where necessary.
- [ ] My changes properly paginate the target service provider's API
- [ ] My changes properly handle rate limiting of the target service provider's
      API
- [ ] My new integration step is instrumented to execute in the correct order
      using `dependsOn`
- [ ] I have referred to the
      [JupiterOne data model](https://github.com/JupiterOne/data-model/tree/main/src/schemas)
      to ensure that any new entities/relationships, and relevant properties,
      match the recommended model for this class of data
- [ ] I have updated the `CHANGELOG.md` file to describe my changes
- [ ] When changes include modifications to existing graph data ingestion, I've
      reviewed all existing managed questions referencing the entities,
      relationships, and their property names, to ensure those questions still
      function with my changes.

### CODE REVIEW

File: src/steps/asset/converter.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function createAssetEntity(asset: RunZeroAsset): Entity {
         updatedAt: parseTimePropertyValue(asset.updated_at, 'ms'),
         alive: asset.alive,
         lastSeen: parseTimePropertyValue(asset.last_seen, 'ms'),
-        lastSeenOn: parseTimePropertyValue(asset.last_seen, 'ms'),
+        lastSeenOn: parseTimePropertyValue(asset.last_seen, 'sec'),
```
Comment (zemberdotnet) [inline src/steps/asset/converter.ts:24]: It looks like the other properties `createdAt`, `updatedAt`, `lastSeen`, and `firstSeen` are all being processed as `ms`. Should we adjust those as well?

Comment (DrewJohnsonGT) [reply src/steps/asset/converter.ts:24]: That is true, this is specifically a follow-up to https://github.com/JupiterOne/graph-rumble/pull/77/files
Which we are trying to align the `lastSeenOn` format with other device integrations.
I don't have context into those other values specifically, but can definitely update them as well if you think that's appropriate

Comment (zemberdotnet) [reply src/steps/asset/converter.ts:24]: The others also look like they are also in seconds. I think it'd be good to just fix them all.

Comment (DrewJohnsonGT) [reply src/steps/asset/converter.ts:24]: Done!

**Review by zemberdotnet — APPROVED** [review]
🚀

**Pull request merged**